app/src/main/res/drawable/ic_down_arrow.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+  ~ Copyright (c) 2020 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="10dp"
+    android:height="5dp"
+    android:viewportWidth="10"
+    android:viewportHeight="5">
+    <path
+        android:fillColor="?attr/brokenSiteOutlinedColor"
```

Comment:
Not sure if we should be using this color here. Pretty generic image vs very concrete color reference.

Reply:
I changed the name of the image, is kind of a weird one because the field is not really a dropdown hence the need for a specific image, but it should only be used in the broken sites screen.

Reply:
why it should be only used from the broken site screen? I mean, it's just a down arrow, we should be able to use it from anywhere we need it, and change the color using `app:tint`, right?